static/scss/homepage.scss

```diff
@@ -52,8 +52,8 @@ body.app-media {
 
       .micromasters-logo {
         height: 23px;
-        width: 152px;
-        top: 10px;
+        width: 252px;
+        top: 8px;
```

Comment:
Just curious: why this change?

Reply:
The width of the logo is different now, so positioning had to adjusted.